docs/guides/medical.md

```diff
@@ -0,0 +1,181 @@
+# ArmaForces - Medical
+
+## Menu Medyczne
+
+### Ustawienie sterowania
+
+Do obsługi Medicala należy wpierw mieć zbindowane menu medyczne. Istnieje możliwość udzielania pierwszej pomocy za pomocą ACE, ale jest to nieporęczne. Na poniższych obrazkach przedstawiono w jaki sposób możemy zbindować szybki dostęp do menu medycznego. 
+
+![Ustawienie bindu menu medycznego](../_data/guides/medical/key_medical_menu.png)
+
+### Obsługa Medicala
+
+Po kliknięciu przypisanego klawisza (domyślnie `H`) wyświetli nam się menu medyczne nasze osoby, na którą mamy skierowany kursor i jest wystarczająco blisko. Jeśli w nikogo nie celujemy wybrana zostaje nasza postać.
+
+![Menu medyczne](../_data/guides/medical/medical_menu.png)
+
+- **Logi aktywności** – Informacje o wykonanych czynnościach np.założenie stazy, zabandażowanie rany, przetoczenie krwi itp.z podaną godziną. 
+- **Szybki podgląd** – Wyniki pomiarów ciśnienia krwi oraz tętna wraz z podaną godziną.
+- **Opis** – Kończyna,ilość utraconej krwi przez organizm, stopień odczuwanego bólu, wykaz ran na danej kończynie lub torsie.
+- **Status** – Graficzny status organizmu. Im barwa danej kończyny jest bardziej czerwona, tym większe poniosła obrażenia. Niebieski kolor oznacza stłuczenie(nie da się usunąć, ale nie wpływa z znaczący sposób na organizm). Niebieskie paski oznaczają założoną stazę na danej kończynie. Możemy kolejno sprawdzać, opatrywać, podawać środki pierwszej pomocy na daną kończynę/tors poprzez kliknięcie na dany fragment na obrazku.
+- **Badanie i leczenie** – W tych zakładkach umieszczone są wszystkie akcje związane z czynnościami pierwszej pomocy. Poniżej przedstawiono opis każdej zakładki, zaczynając od skrajnej lewej.
+	1. **Karta segregacyjna** – Umieszczone są w niej niezbędne informację dotyczące podanych środków medycznych oraz staz, tj. skumulowana liczba oraz czas od ostatniej aplikacji.
+	2. **Zbadaj pacjenta** – Możemy sprawdzić ciśnienie krwi oraz tętno. Tylko medyk jest w stanie dokładnie określić wynik pomiaru, tj. w wartościach liczbowych.
+	3. **Bandaże/złamania** – Wyświetla wszystkie bandaże będące na wyposażeniu twoim lub kolegi, które można użyć do opatrzenia rany oraz opcję założenia/ściągnięcia stazy.
+	4. **Leki** – Wyświetla wszystkie leki będące na wyposażeniu twoim lub kolegi. Adrenalinę i morfinę aplikujemy w kończyny, APAP doustnie.
+	5. **Zaawansowane zabiegi** – Głównie przeznaczone dla medyka. Znajdują tu się akcje wykonania RKO tj. Resuscytacja krążeniowo-oddechowa (opcja dostępna dla wszystkich), zaszycie ran oraz przetoczenie krwi, osocza lub soli fizjologicznej.
+	6. **Ciągnij/Nieś** – Możemy przeciągnąć rannego lub go przenieść na plecach. Wybór metody jest uzależniony od danej sytuacji.
```

Comment:
I can't see an explanation `when do what` for this. It could be useful to tell people that they should just drag the body unless _something_.